test/integration/test_task_run_diagnostic_reports.py

```diff
@@ -68,25 +68,45 @@ def test_run_diagnostic_reports():
 </body>
 </html>"""
 
-    report_1 = "minimal"
-    report_2 = "other"
+    minimal_email_props = {
+        "subject": "VIMC diagnostic report: tid - testGroup - testDisease",
+        "recipients": ["minimal_modeller@example.com", "science@example.com",
+                       "estimate_uploader@example.com",
+                       "estimate_uploader2@example.com"]
+    }
+
+    other_email_props = {
+        "subject": "New version of another Orderly report",
+        "recipients": ["other_modeller@example.com", "science@example.com",
+                       "estimate_uploader@example.com",
+                       "estimate_uploader2@example.com"]
+    }
+
+    minimal_is_first = (result[versions[0]]["report"]) == "minimal"
```

Comment:
Real nitpick, but what is the purpose of these parentheses? Would make more sense if they were used to make precedence explicit:
```suggestion
    minimal_is_first = (result[versions[0]]["report"] == "minimal")
```

Reply:
Oh good spot, I think it probably it got copied over from somewhere else, as I reworked this code a couple of times to sort out the email ordering misery!